src/day2/scan_passwds.rb

```diff
@@ -0,0 +1,100 @@
+#! /usr/bin/env -S ruby -w
+
+#: AoC Day 2 - Count valid passwords. (See `problems.txt` for the exact
+#: problems definitions.)
+
+require 'optparse'
+require 'ostruct'
+
+module ScanPasswds
+
+  extend self
+
+  #: Return true if given passwd matches original shopkeeper's policies
+  #: constraints, false otherwise. (See day2/problems.txt for details.)
+  def match_constraints_orig?(min, max, char, passwd)
```

Comment:
I meant to not abbreviate the name of the variables: `passwd` -> `password`. Min, max and char are fine. Those are widespread enough.

Reply:
I see. fixed in 44e54b6.